backend/benefit/applications/services/ahjo_payload.py

```diff
@@ -0,0 +1,134 @@
+        "Language": "fi",
+        "PublicityClass": "Julkinen",
+        "InternalTitle": f"Avustuksen myöntäminen, työllisyyspalvelut, \
+              työnantajan Helsinki-lisä vuonna {application_year}, \
+              työnantaja {application.company_name}",
+        "Subjects": [
+            {"Subject": "Helsinki-lisät", "Scheme": "hki-yhpa"},
+            {"Subject": "kunnan myöntämät avustukset", "Scheme": "hki-yhpa"},
+            {"Subject": "työnantajat", "Scheme": "hki-yhpa"},
+            {"Subject": "työllisyydenhoito"},
+        ],
+        "PersonalData": "Sisältää erityisiä henkilötietoja",
+        "Reference": application.application_number,
+        "Records": case_records,
+        "Agents": [
+            {
+                "Role": "sender_initiator",
+                "CorporateName": application.company.name,
+                "ContactPerson": application.contact_person,
+                "Type": "ExterOnal",
+                "Email": application.company_contact_person_email,
+                "AddressStreet": application.company.street_address,
+                "AddressPostalCode": application.company.postcode,
+                "AddressCity": application.company.city,
+            }
+        ],
+    }
+    return case_dict
+
+
+def _prepare_record_document_dict(attachment: Attachment) -> dict:
+    """Prepare a documents dict for a record"""
+    # If were running in mock mode, use the local file URI
+    file_url = reverse("ahjo_attachment_url", kwargs={"uuid": attachment.id})
+    hash_value = hash_file(attachment.attachment_file)
+    return {
+        "FileName": f"{attachment.attachment_file.name}",
+        "FormatName": f"{attachment.content_type}",
+        "HashAlgorithm": "sha256",
+        "HashValue": hash_value,
+        "FileURI": f"{settings.API_BASE_URL}{file_url}",
+    }
+
+
+def _prepare_record(
+    record_title: str,
+    record_type: str,
+    acquired: datetime,
+    reference: Union[int, uuid.UUID],
+    documents: List[dict],
+    handler: User,
+    publicity_class: str = "Salassa pidettävä",
+):
+    """Prepare a single record dict for Ahjo."""
+
+    return {
+        "Title": record_title,
+        "Type": record_type,
+        "Acquired": acquired,
+        "PublicityClass": publicity_class,
+        "SecurityReasons": ["JulkL (621/1999) 24.1 § 25 k"],
+        "Language": "fi",
+        "PersonalData": "Sisältää erityisiä henkilötietoja",
+        "Reference": str(reference),
+        "Documents": documents,
+        "Agents": [
+            {
+                "Role": "mainCreator",
+                "Name": f"{handler.last_name}, {handler.first_name}",
+                "ID": handler.ad_username,
+            }
+        ],
+    }
+
+
+def _prepare_case_records(application: Application) -> List[dict]:
+    """Prepare the list of case records"""
+    case_records = []
+    handler = application.calculation.handler
+    main_document_record = _prepare_record(
```

Comment:
So many parameters. Maybe using them named would make easier to know what's what?